locales/de-DE.json

```diff
@@ -401,7 +401,7 @@
     "spoiler_show_less": "Zeige weniger",
     "spoiler_show_more": "Zeige mehr",
     "thread": "Thread",
-    "try_original_site": "Versuche die original Seite"
+    "try_original_site": "Versuche die originale Seite"
```

Comment:
maybe even translate this to "ursprüngliche Seite" (or "Ursprungsseite", don't know the entire context of this)? 
It not about being the original in the sense of "originale" here, right?